scripts/core/backfill_learnings.py

```
@@ -0,0 +1,642 @@
+    global_env = Path.home() / ".claude" / ".env"
+    if global_env.exists():
+        load_dotenv(global_env)
+    opc_env = Path(__file__).parent.parent.parent / ".env"
+    if opc_env.exists():
+        load_dotenv(opc_env, override=True)
+
+
+def list_s3_keys(bucket: str) -> str:
+    """Run ``aws s3 ls`` and return raw stdout. Returns empty string on error."""
+    try:
+        result = subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)